src/NuGet.CommandLine/Common/ExitCodeException.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace NuGet.CommandLine
+{
+    [Serializable]
+    public class ExitCodeException : CommandLineException
```

Comment:
In general, several commands use CommandLineException, I would rather fix the message getting shown when a CommandLineException is thrown instead of creating another exception here.